apps/collabora.sh

```diff
@@ -100,7 +100,7 @@ fi
 
 # Install Collabora docker
 docker pull collabora/code:latest
-docker run -t -d -p 127.0.0.1:9980:9980 -e "domain=$NCDOMAIN" --restart always --cap-add MKNOD collabora/code
+docker run -t -d -p 127.0.0.1:9980:9980 -e "domain=$NCDOMAIN" --restart always --name code --cap-add MKNOD collabora/code
```

Comment:
Maybe "collabora" is better suited. I mean what is "code", some PHP code or what? I know, but users may not. :D

Reply:
I preferred CODE since a lot of users feel tricked with the limitations that "collabora" has, when they are using CODE not Collabora.
Whatever suits best.

Reply:
collabora-de :laughing:

Reply:
OK, fine, let's go with that.

Please make the same changes to onlyoffice.sh, thanks :)